src/OrderFormAcceptanceTests.Steps/Steps/Hooks.cs

```diff
@@ -55,15 +48,13 @@ public async Task AfterScenario()
             var test = objectContainer.Resolve<UITest>();
```

Comment:
Know it's not changed in this PR but can an instance of `UITest` be injected into the constructor rather than resolving it here?

Reply:
I've tried and it causes a dependency loop that can't be resolved